credentials.py

```diff
@@ -0,0 +1,2 @@
+bot_token = "5245951094:AAHQxTNHM0v7weElue3icAjpYtSI7JJhZCc"
```

Comment:
There is a good instrument to work with secret tokens: https://github.com/theskumar/python-dotenv